pkg/kubelet/cm/dra/claiminfo.go

```diff
@@ -26,14 +26,15 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 	"k8s.io/apimachinery/pkg/util/sets"
 	"k8s.io/kubernetes/pkg/kubelet/cm/dra/state"
+	"k8s.io/kubernetes/pkg/kubelet/cm/dra/state/checkpoint/v1alpha1"
```

Comment:
Let's just rename the `state` folder to `checkpoint` and put everything in there, such that we can refer to this as follows:

```suggestion
	checkpoint "k8s.io/kubernetes/pkg/kubelet/cm/dra/checkpoint/v1alpha1"
```

Reply:
done